### ISSUE TITLE
confusion about the projection matrix used

### ISSUE THREAD

**Issue by ChemJeff** [DLR-RM/SingleViewReconstruction#2]
Hi Max,
Can you further explain your code for perspective unproject matrix? It confuses me for a while since it's not seems like in form of some standard projection matrix. Especially the following line:
https://github.com/DLR-RM/SingleViewReconstruction/blob/2e608ff5967acef1bd88189e7529b2f91368d926/SDFGen/geom/math/Transform_i.h#L39
Thanks!

**Comment by themasterlink** [comment]
I sadly do not remember exactly where I got this equation from, but it is also used in this tutorial:

http://www.songho.ca/opengl/gl_projectionmatrix.html

More precise in this picture: 

![](http://www.songho.ca/opengl/files/gl_projectionmatrix_eq20.png)

Be aware that I flipped near and far value, which is the reason why there is no minus in front of it.

But I am sure there would be other solutions to this problem. But I am no expert on intrinsic camera calibration and the forms it can take.
